docs/implementing.html

```diff
@@ -207,13 +207,13 @@ <h2 id="static-header-footer">Static header and footer</h2>
 </tr>
 <tr>
 	<th>Header</th>
-	<td><a href="https://github.com/wet-boew/GCWeb/blob/master/site/pages/gcweb-theme/static-header-footer/bootstrap-3.html#L1-L77" rel="external">Source code, line 1 to 77</a></td>
-	<td><a href="https://github.com/wet-boew/GCWeb/blob/master/site/pages/gcweb-theme/static-header-footer/bootstrap-4.html#L1-L66" rel="external">Source code, line 1 to 66</a></td>
+	<td><a href="https://github.com/wet-boew/GCWeb/docs/static-header-footer/bootstrap-3.html#L1-L77" rel="external">Source code, line 1 to 77</a></td>
```

Comment:
Ends at line 86 now.
```suggestion
	<td><a href="https://github.com/wet-boew/GCWeb/blob/master/docs/static-header-footer/bootstrap-3.html#L1-L86" rel="external">Source code, line 1 to 86</a></td>
```
Similar for footer and for Bootstrap 4.